src/shared/functions.py

```diff
@@ -158,7 +158,7 @@ def write_metadata(
     s3_client.put_object(
         Bucket=s3_bucket_name,
         Key=f"{enums.BucketPath.META.value}/{meta_type}.json",
-        Body=json.dumps(metadata, default=str),
+        Body=json.dumps(metadata, default=str, indent=2),
```

Comment:
unrelated, but since i was manually pretty-ifying json a ton when downloading data from S3, I decided to just write the files out this way.